### PR TITLE
Clarify what routing table shuffling means

### PR DESCRIPTION
**Pull request by nick-giles-neo**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1268.surge.sh

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 1]
```diff
@@ -1419,7 +1419,7 @@ label:enterprise-edition[Enterprise Edition]
 [frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
 |===
 |Description
-a|Enables shuffling of the returned load balancing result.
+a|Vary the order of the entries in routing tables each time one is produced. This means that different clients should select a range of servers as their first contact, reducing the chance of all clients contacting the same server if alternatives are available.
```
Comment (renetapopova) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:1422]: @nick-giles-neo, this change must be reflected in the codebase as well in order to be merged.

**Comment by renetapopova** [comment]
Depends on https://github.com/neo-technology/neo4j/pull/23553

**Comment by nick-giles-neo** [comment]
Core PR merged.

**Comment by nick-giles-neo** [comment]
Additional core PR: https://github.com/neo-technology/neo4j/pull/23570

**Pull request merged**